### ISSUE TITLE
Handle missing/unpublished plugin error

### ISSUE THREAD

**Issue by epelc** [shanalikhan/code-settings-sync#1202]
🐛 **Describe the bug**  

Sync download fails/hangs if you try to download settings which have a plugin that has been unpublished from the vs code website. You have to check the developer tools chrome console to see the error instead of it telling you in sync download log.

🌴 **Occurs On:** [Upload / Download / Help Configuring ] download

📰 **To Reproduce**

Try to sync settings on computer which doesn't have a plugin which has been unpublished.

ie this select line plugin

https://marketplace.visualstudio.com/items?itemName=andreaskian.vscode-select-line
![image](https://user-images.githubusercontent.com/5204642/81109329-ee033a80-8ee7-11ea-99dc-58a85380a6f3.png)

💪 **Expected behavior**

Shouldn't have to open console to view a error about plugin being unpublished.

You should see a friendly error here instead of having to open the developer tools.
![image](https://user-images.githubusercontent.com/5204642/81109525-3d496b00-8ee8-11ea-9306-22f76a3de9a2.png)

📺 **Console Error Log** 
If applicable, add screenshots or paste the console log from developer console of VSCode (Help Menu > Developer Tool - Console Tab). 

![image](https://user-images.githubusercontent.com/5204642/81109274-d7f57a00-8ee7-11ea-80af-9d535c2df5d4.png)

📺  **Additional context**

**Comment by whdeairen** [comment]
me too!

**Comment by TBBle** [comment]
Same problem as #1200, I think.

**Comment by Stanzilla** [comment]
Yeah just noticed this as well, kinda prevents it from every properly completing sync
